content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/saml-sso.md

```diff
@@ -45,6 +39,141 @@ We don’t synchronize user datastores, so make sure that all users:
+    - Under **Set up [Your App]**, copy the required values.
+1. [Assign users](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal-assign-users#assign-a-user-account-to-an-enterprise-application) to the application.
+1. Configure SAML settings in the Cobalt app in **Settings** > **Identity & Access** > **SAML 2.0**.
+    - For these parameters, enter values from Azure AD:
+      - **IdP Issuer URL**: Azure AD Identifier
+      - **IdP Target URL**: Login URL
+      - **IdP Certificate**: Certificate (Base64)
+    - Save the **IdP Relay State** value that appears in red.
+1. In Azure AD, under **Basic SAML Configuration**, paste the **IdP Relay State** value into **Relay State**.
+1. [Test your configuration](https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal-setup-sso#test-single-sign-on).
+
+### Duo
+
+To set up SAML SSO with Duo, read their [documentation](https://duo.com/docs/sso-generic).
+
+| For This Parameter in Cobalt | Enter This Value from Duo |
+|---|---|
+| IdP Issuer URL | Entity ID |
+| IdP Target URL | Single Sign-On URL |
+| IdP Certificate | Certificate |
+
+<br>
+
+| For This Parameter in Duo | Enter This Value from Cobalt |
+|---|---|
+| Entity ID | `https://api.cobalt.io/users/saml/metadata` |
+| Assertion Consumer Service (ACS) URL | `https://api.cobalt.io/users/saml/auth` |
+| Default Relay State | Your `IdP Relay State` from the Cobalt app |
+
+In Duo, complete the **SAML Response** section with:
+
+- **NameID format**: `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`
+- **NameID attribute**: `mail`
+- **Signature algorithm**: `SHA256`
+- **Signing options**: Select **Sign response** and **Sign assertion**.
+- **Map attributes**:
+  - **IdP Attribute**: enter `mail`.
+  - **SAML Response Attribute**: enter `email`.
+
+### Google
+
+For instructions on how to enable SAML SSO with Google, read their [guide](https://support.google.com/a/answer/6087519?hl=en).
+
+| For This Parameter in Cobalt | Enter This Value from Google |
+|---|---|
+| IdP Issuer URL | Entity ID |
+| IdP Target URL | SSO URL |
+| IdP Certificate | Certificate |
+
+<br>
+
+| For This Parameter in Google | Enter this value from Cobalt |
+|---|---|
+| ACS URL | `https://api.cobalt.io/users/saml/auth` |
+| Entity ID | `https://api.cobalt.io/users/saml/metadata` |
+| Start URL | Your `IdP Relay State` from the Cobalt app |
+
+In the Google Admin console, configure the following:
+
+- On the **Service Provider Details** page, leave the **Signed Response** option unselected (default).
+- On the **Attribute Mapping** page, add an attribute `email`, and select **Basic Information** and **Primary Email**.
+
+Once you finish the configuration process, your application for Cobalt appears in the Google Workspace.
+
+### Okta
+
+See [how to configure SAML SSO with Okta](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Cobalt.html).
```

Comment:
Future - but they now require us to host the configuration details, so this article may be deprecated shortly.

Reply:
Perhaps we add the instructions in addition to the link above.